src/index.html

```diff
@@ -64,33 +64,33 @@ <h2 class="headline__subtitle">LET’S CONNECT</h2>
         <ul class="social-stack">
           <li>
             <a>
-              <img src="./assets/icons/twitter.svg" alt="" />
+              <img src="./assets/icons/twitter.svg" alt="twitter icon" />
             </a>
           </li>
           <li>
             <a>
-              <img src="./assets/icons/linkedin.svg" alt="" />
+              <img src="./assets/icons/linkedin.svg" alt="linkedin icon" />
             </a>
           </li>
           <li>
             <a>
-              <img src="./assets/icons/medium.svg" alt="" />
+              <img src="./assets/icons/medium.svg" alt="medium icon" />
             </a>
           </li>
           <li>
             <a>
-              <img src="./assets/icons/github.svg" alt="" />
+              <img src="./assets/icons/github.svg" alt="github icon" />
             </a>
```

Comment:
- [ ] [OPTIONAL] Kindly add links foreach `a` tags

Reply:
thank you for your seggestion ^^